### ISSUE TITLE
Full analysis - Almost no code can be parsed

### ISSUE THREAD

**Issue by gquerret** [Riverside-Software/sonar-openedge#653]
On behalf of AT

Almost all source code lead to this kind of message:
```
[java] ERROR: Error during code parsing for src/procedures/xxx.p at position lib/browser.i:167:6
[java] antlr.NoViableAltException: unexpected token: >
```

**Comment by gquerret** [comment]
Reproduced, comes from `{&*}` directive in preprocessor which seems to incorrectly deal with quoted parameters containing escaped double quotes.

**Comment by gquerret** [comment]
Fixed in 2.3.1
